voice/record-a-call-with-split-audio.py

```diff
@@ -4,38 +4,36 @@
 
```

Comment:
Missing where the variables are meant to be added here - I'm thinking have env variables would be preferable?